src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js

```diff
@@ -66,7 +66,12 @@ const ReportsPartialsCampaigns = ({ location, dependencies: { dopplerApiClient }
               <div className="dp-rowflex">
                 <div className="col-sm-12 col-md-12 col-lg-12">
                   <h3>
-                    <FormattedMessage id="reports_partials_campaigns.header_title" />
+                    <FormattedMessage
+                      id={
+                        'reports_partials_campaigns.header_title_' +
+                        state.campaignSummaryResults.campaignStatus
```

Comment:
I am not sure and it is arguable. 

Would template strings be cleaner than concatenation?

Reply:
changed